src/3rdparty/walkontable/src/overlays.js

```diff
@@ -584,6 +584,29 @@ class Overlays {
 
     return result;
   }
+
+  /**
+   * Update the clones tables class name.
```

Comment:
This may be nitpicking, but the name `updateTableClassName` doesn't look precise enough for me. It synchronizes the class names between the `master` overlay and the other overlays, so maybe something like `syncOverlayTableClassNames`, or something else along these lines? 

Either way, the JSDoc description could be more elaborate, for example something like:

```suggestion
   * Synchronize the class names between the main overlay table and the tables on the other overlays.
```

also, the empty line in the description is probably not needed.